antarest/study/storage/output_service.py

```diff
@@ -0,0 +1,531 @@
+        auto_unzip: bool = True,
+    ) -> Optional[str]:
+        """
+        Import specific output simulation inside study
+        Args:
+            uuid: study uuid
+            output: zip file with simulation folder or simulation folder path
+            params: request parameters
+            output_name_suffix: optional suffix name for the output
+            auto_unzip: add a task to unzip the output after import
+
+        Returns: output simulation json formatted
+
+        """
+        logger.info(f"Importing new output for study {uuid}")
+        study = self.study_service.get_study(uuid)
+        assert_permission(params.user, study, StudyPermissionType.RUN)
+        self.study_service.assert_study_unarchived(study)
+        if not Path(study.path).exists():
+            raise StudyNotFoundError(f"Study files were not found for study {uuid}")
+
+        output_id = self.study_service.storage_service.get_storage(study).import_output(
+            study, output, output_name_suffix
+        )
+        remove_from_cache(cache=self.study_service.cache_service, root_id=study.id)
+        logger.info("output added to study %s by user %s", uuid, params.get_user_id())
+
+        if output_id and isinstance(output, Path) and output.suffix == ArchiveFormat.ZIP and auto_unzip:
+            self.unarchive_output(uuid, output_id, not is_managed(study), params)
+
+        return output_id
+
+    def output_variables_information(
+        self,
+        study_uuid: str,
+        output_uuid: str,
+        params: RequestParameters,
+    ) -> dict[str, list[str]]:
+        """
+        Returns information about output variables using thematic and geographic trimming information
+        Args:
+            study_uuid: study id
+            output_uuid: output id
+            params: request parameters
+        """
+        study = self.study_service.get_study(study_uuid)
+        assert_permission(params.user, study, StudyPermissionType.READ)
+        self.study_service.assert_study_unarchived(study)
+        return get_output_variables_information(
+            self.study_service.storage_service.get_storage(study).get_raw(study), output_uuid
+        )
+
+    def export_output(
+        self,
+        study_uuid: str,
+        output_uuid: str,
+        params: RequestParameters,
+    ) -> FileDownloadTaskDTO:
+        """
+        Export study output to a zip file.
+        Args:
+            study_uuid: study id
+            output_uuid: output id
+            params: request parameters
+        """
+        study = self.study_service.get_study(study_uuid)
+        assert_permission(params.user, study, StudyPermissionType.READ)
+        self.study_service.assert_study_unarchived(study)
+
+        logger.info(f"Exporting {output_uuid} from study {study_uuid}")
+        export_name = f"Study output {study.name}/{output_uuid} export"
+        export_file_download = self.study_service.file_transfer_manager.request_download(
```

Comment:
The file transfer manager must be an explicit dependency of this one, we should not know the study_service uses it